### PR TITLE
Add a new keybinding "W" to collapse the whole tree

### PR DESCRIPTION
**Pull request by be-ndee**

### CODE REVIEW

File: plugin/nvim-tree-startup.lua [hunk 1]
```diff
@@ -36,3 +36,4 @@ if #x > 0 then
   local msg = "Following options were moved to setup, see git.io/JPhyt: "
   require'nvim-tree.utils'.warn(msg..table.concat(x, ", "))
 end
+
```
Comment (kyazdani42) [inline plugin/nvim-tree-startup.lua:39]: uneeded change :)

**Review by kyazdani42 — COMMENTED** [review]
can you add this in the help doc too please ?

**Comment by kyazdani42** [comment]
@be-ndee i think you pushed a wrong commit on this PR (search path)

**Comment by be-ndee** [comment]
> @be-ndee i think you pushed a wrong commit on this PR (search path)

Oh yes... I did mix it up a little bit. I will close this pull request and create two separate pull requests.

**Pull request closed without merging**